test/data/tracker-lists.json

```diff
@@ -1,7 +1,7 @@
 {
     "entityList": {
         "Oath": {
-            "properties": [
+            "domains": [
```

Comment:
how did that work before with this field being wrong?

Reply:
Because we pointed at the whole different class, none of this was testing the 'new' code